sstables/sstables.cc

```diff
@@ -1455,9 +1455,7 @@ future<> sstable::reload_reclaimed_components() {
     sstlog.info("Reloaded bloom filter of {}", get_filename());
 }
 
-// This interface is only used during tests, snapshot loading and early initialization.
-// No need to set tunable priorities for it.
-future<> sstable::load(const dht::sharder& sharder, sstable_open_config cfg) noexcept {
+future<> sstable::load_metadata(sstable_open_config cfg) noexcept {
```

Comment:
Maybe call it load_metadata_no_validation() to alert the reader that something is missing.